contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -51,9 +52,10 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
   uint256 internal s_fallbackLinkPrice; // not in config object for gas savings
   uint96 internal s_ownerLinkBalance;
   uint256 internal s_expectedLinkBalance;
-  address internal s_transcoder;
-  address internal s_registrar;
+  address internal s_transcoder; // use interface as type
+  address internal s_registrar; // cannot use interface as type bc only use case is to check if msg.sender == s_registrar
```

Comment:
nit. "intentionally not using interface as only use case is to check if msg.sender == s_registrar"

Reply:
ah realised this was more of a q. I think it's fine to not have any comment here